source/gsoc/index.rst

```diff
@@ -5,6 +5,13 @@
 Summer of Code for OSUOSL
 =========================
 
+.. note::
+
+   These pages remain as an archive for informational purposes; OSUOSL is not currently
+   participating in Google Summer of Code, but `stay tuned`_ for updates!
+
+.. _stay tuned: https://www.facebook.com/OSUOSL/
```

Comment:
I'd rather we just keep it this and not link back to a Facebook page we don't
really maintain currently.

```suggestion
   participating in Google Summer of Code.
```